tribits/core/package_arch/TribitsAdjustPackageEnables.cmake

```diff
@@ -671,6 +477,35 @@ macro(tribits_disable_parents_subpackages PARENT_PACKAGE_NAME)
 endmacro()
 
 
+# Macro that disables all forward packages that depend on the given packages
+#
+macro(tribits_disable_forward_required_dep_packages PACKAGE_NAME)
+
+  #message("TRIBITS_DISABLE_FORWARD_REQUIRED_DEP_PACKAGES: ${PACKAGE_NAME}")
+
+  if (
+     (NOT ${PROJECT_NAME}_ENABLE_${PACKAGE_NAME})
+     AND
+     (NOT "${${PROJECT_NAME}_ENABLE_${PACKAGE_NAME}}" STREQUAL "")
+     )
```

Comment:
The formatting of this `if()` statement is a little awkward. Any way you could make it a little nicer?

Reply:
@KyleFromKitware, what would you suggest for a multi-line `if ()` statement in CMake?

For multi-line if () statement clauses in CMake, I have moved to the formatting:

```
    if ( <if-clause-line1>
        <if-clause-line2>   # Indented 2 increments
        ...
        <if-clause-lineN>   # Indented 2 increments
      )   # Closing parentheses indented 1 increment to line up with execution statements 
      <execute-statement1>
      <execute-statement2>
      ...
      <execute-statementM>
    endif()
```

If you know the style, it makes it easy to differentiate the different parts of the if statement and avoids too much right drift.

Reply:
That is pretty close to what I did, but I really like having that closing `)` on its own line, lined up with the execute statements.  That makes it easy to see where the if clause ends and the if statements start (at least for me).